lambda/waf-to-splunk-kinesis-data-transformation/index.js

```diff
@@ -1,17 +1,21 @@
 exports.handler = async (event, context) => {
     /* Process the list of records and transform them */
+    const okCount = 0
+    const droppedCount = 0
```

Comment:
these aren't constants, should be `let`

Reply:
Sorry, I originally had the counts in an object and updated the props.